spec/models/loader_orchestrate_spec.rb

```diff
@@ -127,4 +127,24 @@ def verify_data(path)
       end
     end
   end
+
+  context "with non-root user policy" do
+    let(:base_policy_path) { 'empty.yml' }
+    it "loads the minimal policy" do
+      expect(resource_policy).to be
+      verify_data 'base/empty.txt'
+    end
+    it "applies the policy update with non-root user" do
+      status='sucess'
+      begin
+        ENV['CONJUR_ALLOW_USER_CREATION'] = 'false'
+        replace_policy_with 'non_root_user.yml'
+      rescue Exceptions::InvalidPolicyObject => exc
+        status='failure'
+      ensure
+        ENV['CONJUR_ALLOW_USER_CREATION'] = 'true'
+      end
+      expect(status).to eq('failure')
+    end
```

Comment:
Same suggestion as above, let's mock the environment variable and check for the expected exception.

Reply:
removing all this test